libs/island-ui/core/src/lib/ProfileCard/ProfileCard.tsx

```diff
@@ -46,7 +46,7 @@ export const ProfileCard: FC<ProfileCardProps> = ({
       {image && (
         <Box
           className={styles.image}
-          style={{ backgroundImage: `url(${image})` }}
+          style={{ backgroundImage: `url(${image}?w=400)` }}
```

Comment:
I'm not sure if we should use project specific query params inside the core component, but this is definitely something we should be using when defining the image url when using the component. Maybe add a TS comment to the image prop that the recommended image width is 400px?
> to clarify "project specific" there is a chance projects are using different image hosting services and they might have a different api for custom image sizes if they do offer custom image sizes

Reply:
I moved it to the `TeamList` slice instead.